.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,29 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🙋 Get Support for your issues
+    url: https://valoraapp.com/support
+    about: This issue tracker is not for support questions. Please refer to our support page or submit an in-app support ticket.
```

Comment:
Perhaps some very brief instructions for how to report in-app support issues?

Reply:
Adjusted instructions in commit [74d6270](https://github.com/celo-org/wallet/pull/281/commits/74d627050c69ba9ab964631a6b84fbb2cfa149ec)